src/QsCompiler/Tests.Compiler/TestCases/GlobalVerification.qs

```diff
@@ -365,6 +364,20 @@ namespace Microsoft.Quantum.Testing.GlobalVerification {
         }
     }
 
+    operation AllPathsReturnValue13 () : Int {
+        within {
+            return 1;
```

Comment:
What are the semantics for returning from within a `within` block? How does that inline if the `apply` block isn't empty?

Reply:
This specific case actually results in an error : ErrorCode.ReturnStatementWithinAutoInversion.